ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -109,6 +109,74 @@ def monitored(self):
                     if self.getMonitored(datasource):
                         return True
 
+        # 3 check the service class
+        if hasattr(self, 'serviceclass') and 'serviceclass' in self.getRelationshipNames():
```

Comment:
I don't think I fully understand what is going on here.   I'm guessing, though unsure, that this has something to do with existing objects in ZODB that were created before you changed the base class of this class?

It feels to me like they should be migrated over in a migrate script so that everything is sane, but I also might be missing something.   

So I think at a minimum, a comment is needed here explaining it, but possibly a migration script.